data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -143,64 +171,91 @@ def _climate_factor(self) -> bool:
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
-        climate_criteria = (
-            (
-                self.df[
-                    field_names.EXPECTED_BUILDING_LOSS_RATE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.EXPECTED_POPULATION_LOSS_RATE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-        )
 
-        return (
+        climate_eligibility_columns = [
+            field_names.EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD,
+            field_names.EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD,
+            field_names.EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD,
+        ]
+
+        expected_population_loss_threshold = (
```

Comment:
This is still in a different structure than the other factors. In this, you're calcuating `expected_population_loss_threshold` with  `& self.df[field_names.FPL_200_SERIES]` already baked in. In the other factors, the same `<blah>_threshold` does NOT have `& self.df[field_names.FPL_200_SERIES]` baked in. Please be very consistent! Otherwise errors will get introduced quickly.

Reply:
I see now what you mean by this - I thought you merely wanted the variables instantiated. That make sense to me now - thanks.